main.js

```diff
@@ -18,18 +18,18 @@ program
   .usage("[OPTIONS]...")
   .requiredOption("-s, --source <string>", "admin secret for the source DB")
   .requiredOption("-t, --target <string>", "admin secret for the target DB")
-  .requiredOption(
-    "-c, --collection <string>",
-    "the name of the collection to be sync'ed",
-  )
   .requiredOption(
     "-d, --timestamp <number>",
     "the timestamp from which to start syncing",
     parseInt,
   )
   .option(
-    "-i, --index <string>",
-    "[optional] the name of the index to use to sync the collection",
+    "-c, --collections <string...>",
+    "[optional] the list of Collection names to be sync'ed",
```

Comment:
Include the delimiter in the help text.
```suggestion
    "[optional] the comma-delimited list of Collection names to be sync'ed",
```

Reply:
it's not comma delimited. commas will be treated like a character in the collection name.

Reply:
I need to update the README though with proper instructions!